src/seg_utils.py

```diff
@@ -63,6 +65,66 @@ def train_val_split(dataset, parameters):
     return train_loader, val_loader
 
 
+def crop_split_load(images, masks, parameters, qlty_border_weight=0.2):
+    # Standardization
+    images = (
+        images / 255
+    )  # TODO: Revisit the best normalization option we can do, thinking about transferring between sets and train-inference.
+    images = torch.from_numpy(images)
+    masks = torch.from_numpy(masks)
+
+    qlty_window = parameters.qlty_window
+    qlty_step = parameters.qlty_step
+    qlty_border = parameters.qlty_border
+
+    if images.ndim == 3:
+        images = images.unsqueeze(1)
```

Comment:
This looks a bit suspicious. Could you explain when this case occurs and why we are added a dimension at `axis=1` rather than 0?

Reply:
Sure. This statement is likely to be triggered from the current pipeline. The previous step forges all labeled frames into a numpy nd array. Since each slice we pulled from tiled data_client is in shape of eg: (2560, 2560), this array will be 3d, eg: (3, 2560, 2560). qlty will require the input array to be converted to a tensor of shape (N, C, Y, X), so we need to add the channel information and make that conversion, that's why we are doing `axis=1` instead of 0.

In an earlier commit I did use consecutive `unsqueeze(0)`, and ended with a bug of tensor shape (1, 3, 2560, 2560).

Reply:
I see, makes sense.